libraries/tools/kotlin-stdlib-gen/src/templates/Filtering.kt

```diff
@@ -548,6 +548,7 @@ object Filtering : TemplateGroupBase() {
         specialFor(ArraysOfUnsigned) { inlineOnly() }
 
         doc { "Returns a ${f.mapResult} containing only ${f.element.pluralize()} matching the given [predicate]." }
+        sample("samples.collections.Collections.Filtering.filter")
```

Comment:
Have you run stdlib code generation after that? It should affect array `filter` extension too.